doc/pages/Manual.md

```diff
@@ -1216,8 +1218,10 @@ files.
   for path portability of environment variables on Windows.
 
 - <a id="opamfield-extra-files">`extra-files: [ [ <string> <checksum> ] ... ]`</a>:
-  optionally lists the files below `files/` with their checksums. Used
-  internally for integrity verifications.
+  lists the files below `files/` with their checksums. Used
+  internally for integrity verifications. Before opam 2.3, this field was
+  optional, since opam 2.3 this field is now mendatory when wanting to
+  copy files from the `files/` directory.
```

Comment:
```suggestion
  internally for integrity verification. Before opam 2.3, listing files in
  this field was optional, but since opam 2.3 all files must be listed in
  this field or they will not be copied from the `files/` directory.
```